README.textile

```diff
@@ -10,16 +10,17 @@ There are two main ways of running and installing elasticsearch-head
 
 h4. Running as a plugin of Elasticsearch
 
-* @sudo elasticsearch/bin/plugin -install mobz/elasticsearch-head@
+* @sudo elasticsearch/bin/plugin -install mobz/elasticsearch-head@  (ES 1.x)
```

Comment:
You can also specify that this command (with `-install`) works for 0.x as well:
```
/tmp/downloads/elasticsearch-0.90.11 → bin/plugin 
Usage:
    -u, --url     [plugin location]   : Set exact URL to download the plugin from
    -i, --install [plugin name]       : Downloads and installs listed plugins [*
```

Also IMHO `--install` looks better because that's the variant the is listed in help output of the command and `-install` is sort of undocumented feature :)